developer/src/kmc-kmn/src/kmw-compiler/compiler-globals.ts

```diff
@@ -33,20 +45,45 @@ export function setupGlobals(
   FUnreachableKeys = [];
   FCallFunctions = [];
   FFix183_LadderLength = 100; // TODO: option
+  _minimumKeymanVersion = fk.fileVersion;
 }
 
-export function IsKeyboardVersion10OrLater(): boolean {
-  return fk.fileVersion >= KMX.KMXFile.VERSION_100;
+function isKeyboardVersionAutomaticallyDetermined() {
+  return (fk.flags & KMX.KMXFile.KF_AUTOMATICVERSION) == KMX.KMXFile.KF_AUTOMATICVERSION;
 }
 
-export function IsKeyboardVersion14OrLater(): boolean {
-  return fk.fileVersion >= KMX.KMXFile.VERSION_140;
+function verifyMinimumRequiredKeymanVersion(version: KMX.KMX_Version) {
```

Comment:
Can we have a brief documentation of this? I see it sometimes sets `_minimumKeymanVersion`, which might not be expected from a boolean function.

Maybe a more verbose name like `verifyAndSetMinimumRequiredKeymanVersion`?

Reply:
More verbose name is good; see the documentation I added to the wrapper functions.

Reply:
I think I will do this as a follow-up PR once the chain is complete.

Reply:
* #11966